Source/VisualStudio/vsnet2005.project.cst

```diff
@@ -1107,6 +1107,10 @@
 			  <SubType>Code</SubType>
 			</Compile>
 
+			<Compile Include="<%=DALSubFolder%>ExceptionManager.cs">

```

Comment:
Isn't this conditional based on if you are not using ent lib?

Reply:
That would be the cleaner way of writing it :-D